classes/class-wpcom-liveblog-amp.php

```diff
@@ -517,10 +522,7 @@ public static function get_template( $name, $variables = array() ) {
 	 * @return boolean AMP polling request.
 	 */
 	public static function is_amp_polling() {
-		$amp_latest_update_time = filter_input( INPUT_GET, 'amp_latest_update_time', FILTER_SANITIZE_STRING );
-		if ( ! empty( $amp_latest_update_time ) ) {
-			return true;
-		}
-		return false;
+		// phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification
+		return isset( $_GET[ self::AMP_UPDATE_QUERY_VAR ] ); // input var okay.
```

Comment:
The `// input var okay` comment can be removed. This is from the old WPCS whitelisting, but is now covered by the `// phpcs:ignore ...` on the line above.

Reply:
Thanks, good to know. I've sorted that now 👍